lib/main.js

```diff
@@ -257,17 +258,28 @@ api.getPublicKey = function(publicKey, actor, callback) {
  * @param id the ID of the identity to get the PublicKeys for.
  * @param actor the Identity performing the action (if `undefined`, private
  *          keys will be removed from the results).
+ * @param options takes {capability: 'sign'} to return only keys that
+ *          have a private key pairing and thus can perform signing.
  * @param callback(err, records) called once the operation completes.
  */
-api.getPublicKeys = function(id, actor, callback) {
+api.getPublicKeys = function(id, actor, options, callback) {
+  if(typeof options === 'function') {
+    callback = options;
+    options = undefined;
```

Comment:
We usually do `options = {}`. And then below (right above `async.auto`) we'd do `options = options || {};` to ensure options is an object we can query for things.